external/Directory.Build.props

```diff
@@ -1,10 +1,10 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <IntermediateOutputPath>$(IntermediateOutputPath)$(OSGroup)-$(ArchGroup)/</IntermediateOutputPath>
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
     <ContainsPackageReferences>true</ContainsPackageReferences>
     <!-- We need configuration-specific assets files. -->
+    <IntermediateOutputPath>$(IntermediateOutputPath)$(OSGroup)-$(ArchGroup)/</IntermediateOutputPath>
     <RestoreOutputPath>$(IntermediateOutputPath)</RestoreOutputPath>
```

Comment:
Here is another place we are setting RestoreOutputPath is that going to cause any issues?

Reply:
RestoreOutputPath must ALWAYS be in sync with IntermediateOutputPath. If you don't set them accordingly the intermediate restore files will be placed into location $(RestoreOutputPath) but they are searched for in $(IntermediateOutputPath).